src/main/java/com/example/pokebattlez/battle/model/Move.java

```diff
@@ -0,0 +1,20 @@
+package com.example.pokebattlez.battle.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Move {
+    private String name;
+
+    private int position;
+
+    private int power;
+    private int acc;
+    private int pp;
```

Comment:
Even if this is well known abbreviation in the Pokemon world, make this code readable to non-Pokemon devs.  :)